### PR TITLE
Add response headers

### PR DESCRIPTION
**Pull request by heliprokupetz14**

### CODE REVIEW

File: nginx-s3-gateway/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,8 @@ COPY jwt.js /etc/nginx/jwt.js
 COPY 21-envsubst-on-nginx-conf.sh /docker-entrypoint.d
 COPY default.conf.template /etc/nginx/templates
 COPY nginx.conf.template /etc/nginx/nginx.conf.template
+COPY cors.conf.template /etc/nginx/templates/gateway/cors.conf.template
+# COPY s3gateway.js /etc/nginx/include/s3gateway.js
```
Comment (netanelC) [inline nginx-s3-gateway/Dockerfile]: Remove it

File: nginx-s3-gateway/default.conf.template [hunk 2]
```diff
@@ -87,6 +87,8 @@ server {
         # @s3, @s3Listing or @error405.
 
         js_content s3gateway.redirectToS3;
+
```
Comment (netanelC) [inline nginx-s3-gateway/default.conf.template]: Unnecessary

File: nginx-s3-gateway/default.conf.template [hunk 3]
```diff
@@ -140,6 +142,8 @@ server {
 
         proxy_pass ${S3_SERVER_PROTO}://storage_urls$s3uri;
 
+
```
Comment (netanelC) [inline nginx-s3-gateway/default.conf.template:144]: Unnecessary

File: nginx-s3-gateway/default.conf.template [hunk 4]
```diff
@@ -196,7 +200,7 @@ server {
 
         # Comment out this line to receive the error messages returned by S3
         error_page 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 420 422 423 424 426 428 429 431 444 449 450 451 500 501 502 503 504 505 506 507 508 509 510 511 =404 @error404;
-
+
```
Comment (netanelC) [inline nginx-s3-gateway/default.conf.template:201]: Unnecessary

**Review by EllaMartirosyan — CHANGES_REQUESTED** [review]
Should not be merged to master

**Pull request closed without merging**